_posts/2024-03-22-diving-into-jtag-part-3.md

```diff
@@ -0,0 +1,366 @@
+---
```

Comment:
I should have put the line wrapping in a separate commit so changes would be easier to review. A summary of my changes is in the PR description.